lib/manager/pipenv/extract.js

```diff
@@ -43,11 +43,32 @@ function extractFromSection(pipfile, section, registryUrls) {
     return [];
   }
   const specifierRegex = new RegExp(`^${specifierPattern}$`);
-  const deps = Object.entries(pipfile[section])
+  const oneSection = pipfile[section];
+
+  if (section === 'packages') {
+    Object.keys(oneSection).forEach(key => {
+      if (typeof oneSection[key] === 'string') return;
+
+      const isObject =
+        !Array.isArray(oneSection[key]) &&
```

Comment:
Please use the `@sindresorhus/is` package for this instead

Reply:
Done.